app/helpers/data.py

```diff
@@ -204,20 +204,36 @@ def add_session_to_event(request, event_id, state=None, use_current_user=True, n
         """
         form = request.form
         slide_temp_url = form.get('slides_url')
+        slide_custom_url= form.get('slides_custom_url')
+
         video_temp_url = form.get('video_url')
+        video_custom_url= form.get('video_custom_url')
+
         audio_temp_url = form.get('audio_url')
+        audio_custom_url= form.get('audio_custom_url')
+
```

Comment:
IMO there's no problem in showing those urls. You will have to show them that anyways when the user comes back to edit the cfs (i.e when prefilling the url box in edit mode, which is not happening as of now), as there is no distinction b/w a uploaded file url and linked url right now.